spec/system/performance_tracking/viewing_spec.rb

```diff
@@ -13,16 +13,49 @@
         heading_text = page.first('.govuk-heading-xl').text
         expect(heading_text).to eq('Manage users')
       end
+
+      it 'does not show the performance tracking nav link' do
+        expect(page).not_to have_link('Performance tracking')
+      end
+
+      context 'when user manager role is supervisor but without service access enabled' do
```

Comment:
I think this test should explicitly say it is testing the feature flag rather general than system behaviour, maybe even explicitly set the feature flag value to `false`